pkg/network/http/gotls/lookup/internal/generate_luts.go

```diff
@@ -232,22 +232,11 @@ func inspectBinary(binary lutgen.Binary) (interface{}, error) {
 	}
 
 	// Inspect the binary using `binspect`
-	config := bininspect.Config{
-		Functions: []bininspect.FunctionConfig{
-			{
-				Name:                   "crypto/tls.(*Conn).Write",
-				IncludeReturnLocations: false,
-			},
-			{
-				Name:                   "crypto/tls.(*Conn).Read",
-				IncludeReturnLocations: false,
-			},
-			{
-				Name:                   "crypto/tls.(*Conn).Close",
-				IncludeReturnLocations: false,
-			},
-		},
-		StructOffsets: []bininspect.StructOffsetConfig{
+	rawResult, err := bininspect.InspectWithDWARF(elfFile,
+		[]string{"crypto/tls.(*Conn).Write",
```

Comment:
Consider extracting this list to a const

Reply:
good idea